core/arch/arm/dts/stm32mp157c-ed1.dts

```diff
@@ -90,10 +90,14 @@
 		regulator-type = "voltage";
 		regulator-always-on;
 
+//		gpios = <&gpiof 14 (GPIO_STM32_NON_SECURE |GPIO_ACTIVE_HIGH)>;
```

Comment:
Why add this line?

Reply:
My mistake! A leftover from few local tests. I'll remote it. Thanks.